TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/CheckYourAnswers.cshtml.cs

```diff
@@ -26,4 +111,22 @@ public Func<Guid, JourneyInstanceId, string> NextPage()
     {
         return (Id, journeyInstanceId) => LinkGenerator.PersonInduction(Id);
     }
+
+    public override async Task OnPageHandlerExecutionAsync(PageHandlerExecutingContext context, PageHandlerExecutionDelegate next)
```

Comment:
Add validation that state is "Complete" i.e. all fields that need to be are populated